src/assets/database.sql

```diff
@@ -366,6 +366,54 @@ INSERT INTO `commands` (`name`, `description`, `examples`, `tool`, `type`, `comm
 ('Zerocopy', '(client only) Use a "zero copy" method of sending data', 'iperf3 --zerocopy', 19, 'checkbox', '--zerocopy', NULL),
 ('Dont fragment', '(client only) Set the IPv4 Dont Fragment (DF) bit on outgoing packets', 'iperf3', 19, 'checkbox', '--dont-fragment', NULL);
 
+--
+-- Populate data for `commands`: dig
+--
+INSERT INTO `commands` (`name`, `description`, `examples`, `tool`, `type`, `command`, `example`, `sudo`, `category`) VALUES 
+('IPv4 only', 'Use IPv4 query transport only', 'dig -4 example.com', 27, 'checkbox', '-4', NULL, NULL, 'QUERY OPTIONS'),
+('IPv6 only', 'Use IPv6 query transport only', 'dig -6 example.com', 27, 'checkbox', '-6', NULL, NULL, 'QUERY OPTIONS'),
+('addr/port (opt)', 'Bind to source address/port', 'dig -b 10.0.0.1#80', 27, 'input', '-b', NULL, NULL, 'QUERY OPTIONS'), 
+('Query Class', 'The most commonly used class is IN (Internet), which is the default for most DNS queries. However, there are other classes like CH (Chaos) and HS (Hesiod) which are less used', 'dig -c HS example.com', 27, 'input', '-c', NULL, NULL, 'QUERY OPTIONS'),
+('Port Number', 'Specifies the port number for the query', 'dig -p 80 example.com', 27, 'input', '-p', NULL, NULL, 'QUERY OPTIONS'),
+('Query Name', 'Specifies the query name for the DNS lookup', 'dig -q example.com', 27, 'input', '-q', NULL, NULL, 'QUERY OPTIONS'),
```

Comment:
Hey @mayara-santos01 , as I can see you can set `-q` to be the `dig`'s `target` option when adding the tool downthere.

Reply:
this was removed.